modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/dao/JDBCResourceDAO.java

```diff
@@ -376,4 +381,237 @@ public boolean updateResource(String applicationKey, String resourceType, String
         return false;
     }
 
+    /**
+     * Add deployement config details
+     *
+     * @param deploymentConfig deployement config
+     */
+    public void addDeploymentConfig(DeploymentConfig deploymentConfig) throws AppFactoryException {
+        Connection connection = null;
+        PreparedStatement addDeploymentPreparedStatement = null;
+
+        try {
+            connection = AppFactoryDBUtil.getConnection();
+            addDeploymentPreparedStatement = connection.prepareStatement(SQLConstants.ADD_DEPLOYMENT);
+            addDeploymentPreparedStatement.setString(1, deploymentConfig.getDeploymentName());
+            addDeploymentPreparedStatement.setInt(2, deploymentConfig.getReplicas());
+            addDeploymentPreparedStatement.executeUpdate();
+
+            List<Container> containerList = deploymentConfig.getContainers();
+            addContainer(containerList, deploymentConfig.getDeploymentName(), connection);
+            if(addDeploymentPreparedStatement.getUpdateCount() > 0){
```

Comment:
This is fixed in following pull request.
https://github.com/wso2/product-af/pull/800/commits/251406ca0505bcb4e46024e283b181a333ea49be